README.md

```diff
@@ -9,68 +9,301 @@
+### Real-time refactoring suggestions
 
-### Sourcery Pro
+![Refactoring Code with Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_VS_Code_Example_2.gif)
 
-Looking for an even bigger quality boost for your code?
+While you work, Sourcery will review all of the Python files you have open and look for opportunities to clean up and refactor your code. Once Sourcery finds a potential refactoring it will underline that section of your code.
 
-*Automatic method extraction*
+Hover your mouse over the underlined section of code to see the changes Sourcery suggests and see a diff of the proposed change.
 
-Sourcery can find duplicate code or coherent blocks that should be extracted into a method. It will then do the extraction for you - all you need to do is name the method!
+To make the change, just bring up the quick fix menu and choose to accept the change. Sourcery will then apply the change to your code. We're constantly adding new refactorings that Sourcery can make. The current list of the types of refactorings Sourcery can make is available **[here](https://docs.sourcery.ai/refactorings/)**. 
 
-![Extract Method](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Extract_Method_VS_Code_Demo.gif)
+The commands to bring up the quick fix menu depend on your OS & region, but generally they are:
 
-*Multi file analysis & Duplicate analysis (beta):*
 
-Right click on any file or folder in the Explorer and hover over the Sourcery menu to see some of our more advanced features. 
+| VS Code | Mac | Command . |
+| --- | --- | --- |
+| VS Code  | Windows | Ctrl . |
+| VS Code | Linux | Ctrl . |
 
-Choose "Scan with Sourcery" to have Sourcery analyse all of the files in that folder, not just your current folder. 
+Sourcery reviews all of the Python files you have open. You can get an overview of all the suggestions Sourcery has in the Problem window.
 
-![Scan for refactorings](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/VS_Code_Scan_for_Refactorings.gif)
+#### Other Suggested Tools
 
-Or choose "Detect Clones" to have Sourcery scan for duplicate sections of code across those files. 
+- We recommend using Sourcery alongside the [Error Lens plugin](https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens) - it helps make refactoring suggestions easier to see and understand.
 
-![Detect duplicate code](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Duplicate_Code_Detection_VS_Code.gif)
+### Continuous code quality feedback
+
+![Code Quality Metrics in Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_Metrics_VS_Code.gif)
+
+Sourcery gives each of your functions a quality score ranging from 0%(bad) - 100% (good) and also gives you sub-scores on Method Length, Complexity, and Working Memory so that you can figure out how to structure your code as cleanly as possible.
```

Comment:
```suggestion
Sourcery gives each of your functions a quality score ranging from 0% (bad) - 100% (good) and also gives you sub-scores on Method Length, Complexity, and Working Memory so that you can figure out how to structure your code as cleanly as possible.
```